CollAction/Controllers/ProjectsController.cs

```diff
@@ -311,6 +317,25 @@ public IActionResult Embed()
             }
         }
 
+        [Authorize]
+        [HttpGet]
+        public IActionResult ThankYouCommit(string name)
+        {
+            if(name != null)
```

Comment:
Formatting again, should be "if ("

Are you using visual studio? These things should auto-correct.

Reply:
VS code does not show this formatting errors by default

Reply:
Fixed in 0544b69